sklearn/datasets/kddcup99.py

```diff
@@ -347,6 +345,21 @@ def _fetch_brute_kddcup99(subset=None, data_home=None,
 
         joblib.dump(X, samples_path, compress=0)
         joblib.dump(y, targets_path, compress=0)
+
+        # check md5 of dumped samples and targets
```

Comment:
well we'd lose the ability to detect the case where files later get corrupted? or did i misunderstand your original comment?